sn_node/src/node/flow_ctrl/periodic_checks.rs

```diff
@@ -64,130 +64,115 @@ impl PeriodicChecksTimestamps {
             last_dysfunction_check: Instant::now(),
         }
     }
-}
 
-impl FlowCtrl {
     /// Generate and fire commands for all types of periodic checks
-    pub(super) async fn perform_periodic_checks(&mut self) {
-        self.enqueue_cmds_for_standard_periodic_checks().await;
+    pub(super) async fn periodic_checks_cmds(&mut self) -> Vec<Cmd> {
+        let mut cmds = self.enqueue_cmds_for_standard_periodic_checks().await;
 
         if !self.node.read().await.is_elder() {
-            self.enqueue_cmds_for_adult_periodic_checks().await;
+            cmds.extend(self.enqueue_cmds_for_adult_periodic_checks().await);
 
             // we've pushed what we have as an adult and processed incoming msgs
             // and cmds... so we can return already
-            return;
+            return cmds;
         }
 
-        self.enqueue_cmds_for_elder_periodic_checks().await;
+        cmds.extend(self.enqueue_cmds_for_elder_periodic_checks().await);
+
+        cmds
     }
 
     /// Periodic tasks run for elders and adults alike
-    async fn enqueue_cmds_for_standard_periodic_checks(&mut self) {
+    async fn enqueue_cmds_for_standard_periodic_checks(&mut self) -> Vec<Cmd> {
```

Comment:
rename since we no longer enqueue
```suggestion
    async fn standard_periodic_checks(&mut self) -> Vec<Cmd> {
```